### PR TITLE
Make log file option to gc optional

### PR DESCRIPTION
**Pull request by natefoo**
Workaround for https://github.com/cvmfs/cvmfs/issues/3045.

### CODE REVIEW

**Comment by natefoo** [comment]
@hexylena does `block` really have an FQCN? I don't see it in the docs and the linter doesn't think it does.

**Comment by hexylena** [comment]
strong chance it doesn't and I was overzealous and missed it amongst the 20 other Linter errors I low key regret fixing

File: tasks/client.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 - name: Configure CernVM-FS config repository
```
[the remark this answers is not in the document]
Comment (natefoo) [reply tasks/client.yml:29]: This is the error I just fixed, why is it showing up now?

[the remark this answers is not in the document]
Comment (hexylena) [reply tasks/client.yml:29]: yeah we just ignore this one. ansible lint literally does not support blocks at all.

[the remark this answers is not in the document]
Comment (hexylena) [reply tasks/client.yml:29]: yeah we just ignore this one. ansible lint literally does not support blocks at all.

**Pull request merged**